nova/templates/vmware-vspc-service.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Service
+apiVersion: v1
+
+metadata:
+  name: nova-vmware-vspc
+  labels:
+    system: openstack
+    type: backend
+    component: nova
+spec:
+  type: NodePort
+   selector:
+   name: nova-vmware-vspc
```

Comment:
This has to be intended by one. It is an attribute of the selector.
The selector of the nodeport should choose all pods with the name `nova-vmware-vspc`